clients/src/main/java/org/apache/kafka/common/utils/SystemTime.java

```diff
@@ -16,8 +16,11 @@
  */
 package org.apache.kafka.common.utils;
 
+import java.util.concurrent.TimeUnit;
+
 /**
- * A time implementation that uses the system clock and sleep call
+ * A time implementation that uses the system clock and sleep call. Use `Time.SYSTEM` instead of creating an instance
+ * of this class.
  */
 public class SystemTime implements Time {
```

Comment:
I'm happier maintaining compatibility, just wanted to make sure we were doing so knowingly and intentionally.